lib/modules/manager/helmfile/schema.ts

```diff
@@ -11,16 +11,21 @@ export type HelmRepository = z.infer<typeof HelmRepository>;
 export const HelmRelease = z.object({
   name: z.string(),
   chart: z.string(),
-  version: z.string(),
+  version: z
+    .string()
+    .or(z.number())
+    .optional()
+    .nullable()
+    .transform((version) => (version ? String(version) : null)),
```

Comment:
```suggestion
    .transform((version) => (version ? version.toString() : null)),
```
is this really optional or nullable? 🤔

Reply:
I have this optional so we can still return `invalid-version` and nullable so I can set it to null rather than undefined. 

Further this depends how the parser interprets missing values after removing templates. e.g.
```yaml
test: 
```

Reply:
ok, can we still prefer the above suggestion over the existing?

or use a string template? I'm pretty sure it's not a good practice to use constructor functions without `new` call

Reply:
🤷‍♂️